.github/workflows/actions/deploy-environment/action.yml

```diff
@@ -67,11 +73,21 @@ runs:
       id: terraform
       run: |
         make ci ${{ inputs.environment_name }} terraform-apply
-        cd terraform && echo ::set-output name=url::https://$(terraform output -raw auth_server_fqdn)/
+        cd terraform && echo ::set-output name=url::$(terraform output -raw auth_server_fqdn)/
+        echo ::set-output name=postgres_server_name::$(terraform output -raw postgres_server_name)
       env:
         ARM_ACCESS_KEY: ${{ steps.get_secrets.outputs.TFSTATE-CONTAINER-ACCESS-KEY }}
         TF_VAR_azure_sp_credentials_json: ${{ inputs.azure_credentials }}
-        TF_VAR_docker_image: ${{ inputs.docker_image }}
-        TF_VAR_authserver_tag: ${{ inputs.authserver_tag }}
-        TF_VAR_testclient_tag: ${{ inputs.testclient_tag }}
+        IMAGE_TAG: ${{ inputs.image_tag }}
       shell: bash
+
+    - name: Public IP
+      id: ip
+      uses: haythem/public-ip@v1.2
+
+    # - name: Azure CLI script
```

Comment:
Nit: do we need this?

Reply:
Ah missed this. This is not needed at the moment.